config/locales/errors/es.yml

```diff
@@ -38,6 +38,9 @@ es:
       send_link_limited: Ha intentado demasiadas veces, por favor, inténtelo de nuevo
         en %{timeout}. También puede retroceder y elegir utilizar su computadora
         como alternativa.
+    enter_code:
+      rate_limited_html: Ingresó un código de verificación incorrecto demasiadas
+        veces. <br><strong>Inténtelo de nuevo dentro de %{timeout}.</strong>
```

Comment:
Translation issue: The English says "Try again in X minutes" by which we mean "after X minutes." The Spanish here says "within X minutes" which is the reverse of what we mean. The French is more ambiguous, like the English. @artfulaction

Reply:
Thanks for flagging! We have "Inténtelo de nuevo en %{timeout}." from a rate-limited text, so let's use that again here.